corehq/apps/case_importer/util.py

```diff
@@ -207,6 +208,8 @@ def get_importer_error_message(e):
                  'Please choose a file that is not password protected.')
     elif isinstance(e, ImporterExcelError):
         return _("The file uploaded has the following error: {}").format(str(e))
+    elif isinstance(e, ImporterRawError):
```

Comment:
This is purely to replicate the existing somewhat messy error handling from Phase 1 of the upload process.